src/containers/Services/ServiceComponents/VerusIdService/VerusIdLogin.js

```diff
@@ -0,0 +1,19 @@
+import { primitives } from 'verusid-ts-client'
+import { SET_DEEPLINK_DATA } from "../../../../utils/constants/storeType";
+
+export const processVerusId = async (props, loginConsentRequest, extraParams = null) => {
```

Comment:
Please run files like these through a code formatter like prettier to avoid the spacing issues here, like the ending bracket being indented on a different level than the first line of the function (especially if its a newly created file so code formatting would not add new deltas).